src/index.html

```diff
@@ -4,9 +4,372 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="moyo"
+        >
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="nav-hover">Laptops & computers </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div
+         class="card cards__card"
+         data-qa="card"
+        >
```

Comment:
Should be 2 spaces everywhere for attributes
```suggestion
        <div
          class="card cards__card"
          data-qa="card"
        >
```